plugins/modules/redhat_subscription.py

```diff
@@ -19,6 +19,16 @@
       registering using D-Bus if possible.
 author: "Barnaby Court (@barnabycourt)"
 notes:
+    - |
+      The module tries to use the D-Bus C(rhsm) service (part of C(subscription-manager))
+      to register, starting from community.general 6.5.0: this is done so credentials
+      (username, password, activation keys) can be passed to C(rhsm) in a secure way.
+      C(subscription-manager) itself gets credentials only as arguments of command line
+      parameters, which is I(not) secure, as they can be easily spoofed by checking the
```

Comment:
```suggestion
      parameters, which is I(not) secure, as they can be easily stolen by checking the
```
unless you are using "spoofed" in a way that I am not familiar with

Reply:
ah, thanks!